brambling/static/brambling/sass/modules/_list-item.sass

```diff
@@ -0,0 +1,5 @@
+.list-item
+	display: flex
+	flex-direction: row
+	justify-content: space-between
+	align-items: center
```

Comment:
Also gonna make give an honorable mention to `display: table-row` and `display: table-cell` which have been able to do many of the flexy things we love with wide browser support for ages, though sometimes with wonky side effects. Not sure it fits in this context, but it looks like it might.